src/gen_apple/ga_dev_gps.cpp

```diff
@@ -0,0 +1,86 @@
+#include "ga_dev_gps.h"
+
+#define GPSECHO  false //Define true to echo gps data to serial monitor
+
+SoftwareSerial mySerial(_PIN_GPS_TX, _PIN_GPS_RX);
+Adafruit_GPS GPS(&mySerial);
+
+void ga_dev_gps_open(void) {
+
+    pinMode(_PIN_GPS_ENABLE , OUTPUT);
+    //Enable pin needs to be pulled low;
```

Comment:
The comment here says the enable pin needs to be pulled low, but the digitalWrite call below pulls it HIGH.